packages/story-editor/src/components/canvas/karma/quickActions.karma.js

```diff
@@ -124,17 +124,20 @@ describe('Quick Actions integration', () => {
     });
 
     it(`clicking the \`${ACTIONS.INSERT_TEXT.text}\` button should select the background, open the text tab in the library and insert the default text`, async () => {
-      // click quick menu button
-      await fixture.events.click(
-        fixture.editor.canvas.quickActionMenu.insertTextButton
-      );
-      expect(fixture.editor.canvas.framesLayer.frames.length).toBe(2);
-      expect(
-        fixture.editor.sidebar.designPanel.selectionSection
-      ).not.toBeNull();
-      expect(document.activeElement).toEqual(
-        fixture.editor.canvas.framesLayer.frames[1].node
-      );
+      await waitFor(async () => {
```

Comment:
You probably only need the `waitFor` around the `expect(fixture.editor.canvas.framesLayer.frames.length).toBe(2);` assertion. `waitFor` is not meant to be wrapped around the whole test.

Please look at other tests for precedence/inspiration.

Reply:
The failing assertion is actually - 
```js
expect(document.activeElement).toEqual(
    fixture.editor.canvas.framesLayer.frames[1].node
);
```
I did try wrapping this in `waitFor`, but it still failed. Another way was to use `fixture.events.sleep(100)` after `click()`, but it is essentially the same as using `setTimeout`.

Reply:
Going with `fixture.events.sleep(100)` for now